### DISCUSSION TITLE
vs code not paired with GitHub desktop

### DISCUSSION THREAD

**Discussion by MatthewProjects** [community/community#42014] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I just moved vs code to my application folder and it’s still not recognizing it
15:42
[https://stackoverflow.com/questions/67780347/i-am-trying-to-update-visual-studio-on-a-mac-to-the-latest-version-but-i-got-err](https://stackoverflow.com/questions/67780347/i-am-trying-to-update-visual-studio-on-a-mac-to-the-latest-version-but-i-got-err?ck_source=careerkarma-web)
15:41
[https://stackoverflow.com/questions/65422670/why-doesnt-vs-code-auto-update-on-macos](https://stackoverflow.com/questions/65422670/why-doesnt-vs-code-auto-update-on-macos?ck_source=careerkarma-web)
15:40
[https://github.com/sparkle-project/Sparkle/issues/936](https://github.com/sparkle-project/Sparkle/issues/936?ck_source=careerkarma-web)
15:38
[https://naveendhanaraj.wordpress.com/2021/07/14/cannot-update-while-running-on-a-read-only-volume-the-application-is-on-a-read-only-volume-please-move-the-application-and-try-again-if-youre-on-macos-sierra-or-later-youll-need-to-move-the-ap/](https://naveendhanaraj.wordpress.com/2021/07/14/cannot-update-while-running-on-a-read-only-volume-the-application-is-on-a-read-only-volume-please-move-the-application-and-try-again-if-youre-on-macos-sierra-or-later-youll-need-to-move-the-ap?ck_source=careerkarma-web)
15:33

**Comment by SilasRodrigues19** [comment]
Hi @MatthewProjects 

There are a few steps you can try to resolve the issue:

1. Make sure that you have installed the latest version of both VS Code and GitHub Desktop on your computer.

2. Check that you have the "GitHub" extension installed in VS Code. To do this, open VS Code, click on the "Extensions" icon on the left sidebar, and search for "GitHub". If the extension is not installed, click "Install" to install it.

3. In VS Code, open the Command Palette (Ctrl+Shift+P on Windows or Command+Shift+P on Mac) and type "Git: Clone" to open the "Git: Clone" command.

4. In the "Repository to clone" field, enter the URL of the repository you want to clone and click "Clone".

5. After the repository is cloned, open the Command Palette again and type "Git: Open Repository in GitHub Desktop" to open the repository in GitHub Desktop.

6. In GitHub Desktop, click on the "File" menu and select "Add Local Repository".

7. In the "Add Local Repository" dialog, navigate to the location of the repository you just cloned in VS Code and click "Add Repository".

8. The repository should now be paired with both VS Code and GitHub Desktop.

**Reply by MatthewProjects** [reply]
when you're talking about the e "GitHub" extension ... do you mean this one? because I did recently enable the GitHub  pull requests and issues....<img width="793" alt="Screen Shot 2022-12-17 at 10 14 57 PM" src="https://user-images.githubusercontent.com/118500031/208282800-73a36276-9ae6-4b6e-92aa-d4c41ab13bf2.png">

or this one 
<img width="774" alt="Screen Shot 2022-12-17 at 10 15 48 PM" src="https://user-images.githubusercontent.com/118500031/208282847-c0ee2b5f-5e7d-47f9-8057-62f04928f70c.png">

**Reply by MatthewProjects** [reply]
<img width="754" alt="Screen Shot 2022-12-17 at 10 24 34 PM" src="https://user-images.githubusercontent.com/118500031/208282948-a04c7121-bf11-436b-85c7-11ede6bbf3e6.png">


it says to use the GitHub pull requests and issues because the "GitHub" extension is not in service  etc....

**Comment by umar-khan25** [comment]
Just open you're VS Code from here you're good to go.
![Screenshot_16](https://github.com/community/community/assets/76748183/050d1d99-4024-497d-999d-172a392156f9)

**Reply by michaelnicol** [reply]
Thank you,

I had an issue where Visual Studio Code and GitHub Desktop became unpaired. Visual Studio had lost all recent changes to most of my branches, even if it was committed.

Opening in VS Code via that button gave me an updated version of my code.

**Comment by hankqqu** [comment]
https://stackoverflow.com/questions/67780347/i-am-trying-to-update-visual-studio-on-a-mac-to-the-latest-version-but-i-got-err?ck_source=careerkarma-web
